mmdeploy/codebase/mmpose/models/backbones/litehrnet.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+import torch.nn.functional as F
+
+from mmdeploy.core import FUNCTION_REWRITER
+
+
+@FUNCTION_REWRITER.register_rewriter(
+    'mmpose.models.backbones.litehrnet.CrossResolutionWeighting.forward')
+def cross_resolution_weighting__forward(ctx, self, x):
+    """Rewrite ``forward`` for default backend.
+
+    Rewrite this function to support export ``adaptive_avg_pool2d``.
+
+    Args:
+        x (list): block input.
+    """
+
+    mini_size = [int(_) for _ in x[-1].shape[-2:]]
```

Comment:
Did that means liteHRNet only support static shape?

Reply:
Yes. We assume all MMPose models do not need to support dynamic shape.